index.html

```diff
@@ -417,8 +423,16 @@ <h2><code>PointerEvent</code> Interface</h2>
 
                 <p>The <dfn><code>PointerEventInit</code></dfn> dictionary is used by the <dfn><code>PointerEvent</code></dfn> interface's constructor to provide a mechanism by which to construct untrusted (synthetic) pointer events. It inherits from the {{MouseEventInit}} dictionary defined in [[UI-EVENTS]]. The steps for constructing an event are defined in [[DOM]]. See the <a href="#examples" title="examples">examples</a> for sample code demonstrating how to fire an untrusted pointer event.</p>
 
-                <div class-"note">Pointer Events include two complementary sets of attributes to express the orientation of a transducer relative the X-Z plane: <dfn><code>tiltX</code></dfn> / <dfn><code>tiltY</code></dfn> (introduced in the original Pointer Events specification), and <dfn><code>altitudeAngle</code></dfn> / <dfn><code>azimuthAngle</code></dfn> (adopted from the <a href="https://w3c.github.io/touch-events/">Touch Events - Level 2</a> specification).
-                    Implementations MUST support both sets of attributes. When an untrusted (synthetic) Pointer Event is created programmatically using the constructor, and only one set of values is provided, the complementary set of attributes MUST be calculated and initialized.</div>
+                <div class="note">
+                    Pointer Events include two complementary sets of attributes to express the orientation of a transducer relative to the X-Y plane: <dfn><code>tiltX</code></dfn> / <dfn><code>tiltY</code></dfn> (introduced in the original Pointer Events specification), and <dfn><code>altitudeAngle</code></dfn> / <dfn><code>azimuthAngle</code></dfn> (adopted from the <a href="https://w3c.github.io/touch-events/">Touch Events - Level 2</a> specification).
+                    Implementations MUST support both sets of attributes. When an untrusted (synthetic) Pointer Event is created programmatically using the constructor, and only one set of attributes is provided, the complementary set of attributes MUST be calculated and initialized by the user agent. If both sets of attributes are provided, no calculation should be performed.
+
+                    <p>
+                    If <code>azimuth</code> and <code>altitude</code> are provided, the user agent MUST initialize <code>tiltX</code>, <code>tiltY</code>
+                    with the calculated values rounded to the nearest integer value. For values falling exactly midway between integers, the user agend MUST always round up.
+                    </p>
+                    <p>If only the value of one of the two attributes is provided, the other attribute MUST be initialized to the default value.</p>
```

Comment:
So does this line refer to both when out of 4 only 1 is present and when out of 4 only 3 is present?

Reply:
> So does this line refer to both when out of 4 only 1 is present and when out of 4 only 3 is present?

My intention was to cover both cases. Basically I would like to apply formulas only in these 2 cases:
- `(hasTiltX || hasTiltY) && !hasAzimuthAngle && !hasAltitudeAngle`
- `(hasAzimuthAngle || hasAltitudeAngle) && !hasTiltX && !hasTiltY`

If `tiltX` and `tiltY` and one of `azimuthAngle`/`altitudeAngle` values are provided, I cannot guarantee that `azimuthAngle`/`altitudeAngle` value as provided by the user matches the formula with regards to `tiltX` and `tiltY`. Same goes if `azimuthAngle` and `altitudeAngle` and one of `tiltX`/`tiltY` values are provided.

My assumption is that the user agent is not going to overwrite a value provided by the user. If the user provides values that are not correct it is an authoring problem.